src/gui/plugins/component_inspector/Noise.qml

```diff
@@ -0,0 +1,424 @@
+/*
+ * Copyright (C) 2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+import QtQuick 2.9
+import QtQuick.Controls 1.4
+import QtQuick.Controls 2.2
+import QtQuick.Controls.Material 2.1
+import QtQuick.Dialogs 1.0
+import QtQuick.Layouts 1.3
+import QtQuick.Controls.Styles 1.4
+import "qrc:/ComponentInspector"
+import "qrc:/qml"
+
+// Item displaying noise information.
+Rectangle {
+  id: noise
+  height: noiseContent.height
+  color: "transparent"
+
+  // Mean value
+  property double meanValue: 0.0
```

Comment:
We should support units where appropriate according to the sensor type

Reply:
Yeah, I think i'll have to add a new property field to the noise qml.

Reply:
Captured in #1232